### DISCUSSION TITLE
Inverting search direction

### DISCUSSION THREAD

**Discussion by JaSpa** [ggandor/leap.nvim#101] [Q&A]
What I'm missing switching over from Lightspeed is the possibility to switch the search direction with `<TAB>` when I accidentally hit `s` instead of `S`.

Is it possible to configure this somehow? Or could there maybe be an additional `special_keys` entry?

**Comment by ggandor** [comment]
No, it's not possible to configure here. 

> Or could there maybe be an additional special_keys entry?

I don't see much value in it (it was added to Lightspeed as a poor man's substitute for bidirectional search, before it was available). Just press `<esc>S` on those rare occasions. :)
